data/sql/updates/pending_db_world/rev_1601277180341917800.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1601277180341917800');
+
+-- Immune to Spell IDs: 9484 / 3355 / 20066 / 57546 / 60192 / 339 / 33786
+UPDATE `creature_template` SET mechanic_immune_mask='550187859' WHERE  entry=34607; -- ToC 10-man Normal Raid
+UPDATE `creature_template` SET mechanic_immune_mask='550187859' WHERE  entry=34648; -- ToC 25-man Normal Raid 
+UPDATE `creature_template` SET mechanic_immune_mask='550187859' WHERE  entry=35655; -- ToC 10-man Heroic Raid
+UPDATE `creature_template` SET mechanic_immune_mask='550187859' WHERE  entry=35656; -- ToC 25-man Heroic Raid
```

Comment:
you should not pass strings where numbers are expected

Reply:
Also dont override flags, add them using mim=mim|flag

Mim = mechanic immune mask